napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -239,6 +242,7 @@ def _radio_button(
         """
         action_name = f'napari:{action_name}'
         btn = QtModeRadioButton(layer, btn_name, mode, **kwargs)
+        btn.setObjectName(f'{btn_name}_button')
```

Comment:
What is the benefit here of setting the object name? I know it's generally good practice, but was this affecting tests before?

Reply:
ops. This is my debug code, where I try to detect which widgets are leaking. 
```suggestion
```